app/shared/accessibility-shortcuts/AccessibilityShortcuts.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import injectT from '../../i18n/injectT';
+
+function AccessibilityShortcuts({ t, mainContentId }) {
+  const mainContentHref = `#${mainContentId}`;
+
+  return (
+    <div className="accessibility-shortcuts">
+      <a
+        className="sr-only skip-link"
```

Comment:
Bad naming, use BEM syntax (check how className been name around app)

If this is a link use Link component from react router dom

Reply:
Renamed these.

In this context we are not linking to another page, but just to an ID within the same page.